packages/webapp/pages/tags/index.tsx

```diff
@@ -123,7 +124,7 @@ const TagsPage = ({
   return (
     <>
       <NextSeo {...seo} />
-      <main className="flex flex-col gap-4 p-0 tablet:px-10 tablet:py-6">
+      <PageWrapperLayout className="flex flex-col gap-4">
```

Comment:
Same here does it need the gap etc?

Reply:
I can move the `flex flex-col gap-4` to a child element, but it is needed for the page layout 👀